admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
+| `currency`  |  yes | [CurrencyEnum](#currencyenum) | The currency of the account. |
+| `isActive`  |  yes | [Integer(1)](#integer) | A flag to indicate whether or not the ledger account is active. Possible values are `1` and `0`. |
+| `createdDate`  |  yes | [DateTime](#datetime)  | Date and time when the ledger account was created. |
+| `createdBy`  |  yes | [String](#string) | The entity that created the ledger account. |
+
+### Limit
+
+Data model for the complex type **Limit**.
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `type`  |  yes | [String](#string) | Limit type. |
+| `value`  |  yes | a positive [Number](#number) | Limit value. |
+
+### Money
+
+For details, see section [7.4.10 Money](../fspiop-api/documents/API-Definition_v1.1.md#7410-money) in the Mojaloop FSPIOP API Definition.
+
+### Participant
+
+Data model for the complex type **Participant**.
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `name`  |  yes | [String](#string) | The name of the participant. |
+| `id`  |  yes | [String](#string) | The identifier of the participant in the form of a fully qualified domain name combined with the participant's `fspId`. |
+| `created`  |  yes | [DateTime](#datetime)  | Date and time when the participant was created. |
+| `isActive`  |  yes | [Integer(1)](#integer) | A flag to indicate whether or not the participant is active. Possible values are `1` and `0`. |
+| `links`  |  yes | [Self](#self) | List of links for a Hypermedia-Driven RESTful Web Service. | 	
+| `accounts`  |  yes | [Accounts](#accounts) | The list of ledger accounts configured for the participant. |
+
+### ParticipantFunds
+
+Data model for the complex type **ParticipantFunds**.
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `transferId`  | yes  | [UUID](#uuid) | Transfer identifier. |
+| `externalReference`  |  yes | [String](#string) | Reference to any external data, such as an identifier from the settlement bank.  |
+| `action`  |  yes | [Enum](#enum) | The action performed on the funds. Possible values are: `recordFundsIn` and `recordFundsOutPrepareReserve`. |
+| `reason`  |  yes | [String](#string) | The reason for the FundsIn or FundsOut action. |
+| `amount`  | yes  | [Money](#money) | The FundsIn or FundsOut amount. |
+| `extensionList`  | no  | [ExtensionList](#extensionlist) | Additional details. |
+
+### ParticipantLimit
+
+Data model for the complex type **ParticipantLimit**.
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `type`  |  yes | [String](#string) | The type of participant limit (for example, `NET_DEBIT_CAP`.)  |
+| `value`  |  yes | [Number](#number) | The value of the limit that has been set for the participant.  |
+| `alarmPercentage`  |  yes | [Number](#number) | An alarm notification is triggered when a pre-specified percentage of the limit is reached. Specifying an `alarmPercentage` is optional. If not specified, it will default to 10 percent. |

```

Comment:
Is this a percentage expressed between 0-1? Or between 0->100?

e.g. does 0.5 mean 50%, or 0.5%?

Reply:
I believe it is between 0-100 but I'm not sure. @mdebarros, please can you help me out here? I will add this info to the doc once I have the answer.